Shared/Model/DB/AppDatabase.swift

```diff
@@ -35,26 +35,21 @@ final class AppDatabase {
         migrator.eraseDatabaseOnSchemaChange = true
         #endif
         
-        migrator.registerMigration("createInitialTables") { db in
-            // Create a table
-            // See https://github.com/groue/GRDB.swift#create-tables
+        migrator.registerMigration("createInitialTables2023") { db in
             
-            try db.create(table: "fundraisingEvent") { t in
-                t.column("id", .blob).primaryKey()
+            // New version of fundraisingEvent
+            try db.create(table: "teamEvent") { t in
+                t.column("publicId", .blob).primaryKey()
                 t.column("name", .text).notNull()
-                t.column("slug", .text).notNull()
-                t.column("amountRaisedCurrency", .text).notNull()
-                t.column("amountRaisedValue", .text)
-                t.column("colors", .text).notNull()
-                t.column("description", .text)
+                t.column("description", .blob).notNull()
                 t.column("goalCurrency", .text).notNull()
                 t.column("goalValue", .text).notNull()
-                t.column("causePublicId", .numeric).notNull().unique()
-                t.column("causeName", .text).notNull()
-                t.column("causeSlug", .text).notNull()
-                t.uniqueKey(["slug", "causeSlug"])
+                t.column("goalNumericalValue", .double).notNull()
+                t.column("totalRaisedCurrency", .text).notNull()
+                t.column("totalRaisedValue", .text).notNull()
+                t.column("totalRaisedNumericalValue", .double).notNull()
             }
-            
+                        
             try db.create(table: "campaign") { t in
                 t.column("id", .blob).primaryKey()
                 t.column("name", .text).notNull()
```

Comment:
My thoughts were that I didn't have a great place to test a migration from the old state, and that I wasn't sure exactly how I'd end up changing the tables until I was done with the rewrite—I was thinking it would be much easier to just start from scratch with a new database?

Reply:
> My thoughts were that I didn't have a great place to test a migration from the old state, and that I wasn't sure exactly how I'd end up changing the tables until I was done with the rewrite—I was thinking it would be much easier to just start from scratch with a new database?

That's a fair point 🤔. Perhaps start with a migration that drops the old tables if they're present?

Reply:
That is a great idea. The latest commit does exactly that, if you want to take a look?

Reply:
Yup, that looks much cleaner!